src/database/build.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const buildDatabase = () => {
+  const connection = require('./dbConnection');
```

Comment:
I think you dont need to make function "buildDatabase" :+1:

Reply:
i exported directly and delete the name ..